src/System.IO.Compression.Brotli/System.IO.Compression.Brotli.csproj

```diff
@@ -47,14 +47,10 @@
     <Compile Remove="System\Interop\Library.OSX.cs" />
   </ItemGroup>
   <ItemGroup>
-    <Content Include="$(BrotliPath)\**">
```

Comment:
All the markup above for setting SourceArch, SourceSystem, and BrotliPath can also be removed.